ansible/roles/lxc/tasks/main.yml

```diff
@@ -340,7 +340,7 @@
   lineinfile:
     path: '/var/lib/lxc/{{ item.name | d(item) }}/config'
     regexp: '^lxc\.mount\s+=\s+'
-    line: 'lxc.mount = /var/lib/lxc/{{ item.name | d(item) }}/fstab'
+    line: 'lxc.mount.fstab = /var/lib/lxc/{{ item.name | d(item) }}/fstab'
```

Comment:
This change should probably be versioned, LXC 3.x supports the `lxc.mount` variant, and 4.x supports the `lxc.mount.fstab` variant. Also, you might need to update the regexp accordingly.

Reply:
Ack. I will try to come up with some useful suggestion.